native/SampleApps/RestAPIExplorer/RestAPIExplorer/Classes/AppDelegate.m

```diff
@@ -48,9 +48,9 @@ - (id)init
     self = [super init];
     if (self) {
         #if defined(DEBUG)
-            [SFLogger sharedLogger].logLevel = SFLogLevelDebug;
+            [SFSDKLogger sharedDefaultInstance].logLevel = DDLogLevelDebug;
```

Comment:
I discussed with Urvin. The default we want in release mode is `Error` and debug mode is `Debug` for all loggers. I should remove these from here.